core/src/data/value/mod.rs

```diff
@@ -902,4 +962,27 @@ mod tests {
             .into()),
         );
     }
+
+    #[test]
+    fn test_unary_minus() {
```

Comment:
unit tests already include the meaning of `test`. so, how about to use simply.. `unary_minus()` for this test name?